dev-qt/qtwebengine/qtwebengine-6.8.3-r1.ebuild

```diff
@@ -0,0 +1,350 @@
+# Copyright 2021-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{11..13} )
+PYTHON_REQ_USE="xml(+)"
+inherit check-reqs flag-o-matic multiprocessing optfeature
+inherit prefix python-any-r1 qt6-build toolchain-funcs
+
+DESCRIPTION="Library for rendering dynamic web content in Qt6 C++ and QML applications"
+SRC_URI+="
+	https://dev.gentoo.org/~ionen/distfiles/${PN}-6.8-patchset-7.tar.xz
+"
+
+if [[ ${QT6_BUILD_TYPE} == release ]]; then
+	KEYWORDS="amd64 arm64"
+fi
+
+IUSE="
+	accessibility +alsa bindist custom-cflags designer geolocation
+	+jumbo-build kerberos opengl +pdfium pulseaudio qml screencast
+	+system-icu vaapi vulkan webdriver +widgets
+"
+REQUIRED_USE="
+	designer? ( qml widgets )
+	test? ( widgets )
+"
+
+# dlopen: krb5, libva, pciutils, udev
+RDEPEND="
+	app-arch/snappy:=
+	dev-libs/expat
+	dev-libs/libevent:=
+	dev-libs/libxml2:=[icu]
```

Comment:
Can drop qtwebengine, given it's a long build I think we should special-case it and I instead did commit 85ef7af8f3134a8d9cad7498965abec94a5833aa instead.